devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -101,5 +142,172 @@ protected SpanByte SliceGenericBuffer(int start, int length)
 
             return _genericBuffer;
         }
+
+
+        /// <summary>
+        /// Draws a pixel on the screen.
+        /// </summary>
+        /// <param name="x">The x coordinate on the screen.</param>
+        /// <param name="y">The y coordinate on the screen.</param>
+        /// <param name="useWhite">Indicates if color to be used, default yes unless is inverted screen.</param>
+        public void DrawPixel(int x, int y, bool useWhite = true)
+        {
+            if ((x >= Width) || (y >= Height))
+                return;
+
+            // x specifies the column
+            int idx = x + (y / 8) * Width;
+
+            if (useWhite)
+                _genericBuffer[idx] |= (byte)(1 << (y & 7));
+            else
+                _genericBuffer[idx] &= (byte)~(1 << (y & 7));
+
+            //INVERSE COLOR: _genericBuffer[idx] ^= (byte)(1 << (y & 7));
```

Comment:
please remove any dead code that is not use. If you really want to keep dead code, please add a comment why. This helps getting everything in a clean state